components/UseCases.js

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image'
+import imageTop from '../assets/img/user-case-privacy-phone.png';
```

Comment:
The images shouldn't be imported inside of the component - but from the page displaying the component